app/lib/surface/Lank.coffee

```diff
@@ -656,8 +656,8 @@ module.exports = Lank = class Lank extends CocoClass
       d.toggle debug
       d.updatePosition()
 
-  addLabel: (name, style) ->
-    @labels[name] ?= new Label sprite: @, camera: @options.camera, layer: @options.textLayer, style: style
+  addLabel: (name, style, labelOptions) ->
```

Comment:
To keep this api backward compatible should there be an explicit default value for labelOptions such as `{}`.

Is this also a good time to add documentation to this method explaining some of the important `labelOptions` that can be passed in with their reasonable defaults?

Reply:
new Label has a default value for labelOptions. Agreed that it would be better to put a default value here, however not in definition but in the body, because putting a mutable default value in argument syntax can be a problem (common error of newbies in Python)

Reply:
Looks like in Coffee script this is not the same problem with default values as in python so it's ok to use `={}` as default.